go/stellar/wallet_state.go

```diff
@@ -444,6 +479,7 @@ func (a *AccountState) refresh(mctx libkb.MetaContext, router *libkb.NotifyRoute
 	seqno, err := a.remoter.AccountSeqno(mctx.Ctx(), a.accountID)
 	if err == nil {
 		a.Lock()
+		// XXX check pending here too?
```

Comment:
Is this a TODO?

Reply:
i thought i pushed a commit to remove that comment...nope, i didn't.  here it is.